src/rabbit_peer_discovery.erl

```diff
@@ -156,10 +162,7 @@ maybe_inject_randomized_delay() ->
 -spec inject_randomized_delay() -> ok.
 
 inject_randomized_delay() ->
-    {Min, Max} = case randomized_delay_range_in_ms() of
-                     {A, B} -> {A, B};
-                     [A, B] -> {A, B}
```

Comment:
@michaelklishin I'm curious why this `case ... end` existed to clean up the values in the first place? Did this have to do with configuration values?

Reply:
Possibly to handle `rabbitmq-autocluster` values or because we've seen someone try to use a list instead of a tuple in the classic/advanced config file. I think by now we can support tuples only, that's what [Cuttlefish produces](https://github.com/rabbitmq/rabbitmq-server/blob/master/priv/schema/rabbit.schema#L882) and the docs only provide new style config file examples.